code/controllers/subsystem/explosions.dm

```diff
@@ -556,14 +556,14 @@ ADMIN_VERB(check_bomb_impacts, R_DEBUG, "Check Bomb Impact", "See what the effec
 				listener.playsound_local(epicenter, null, far_volume, TRUE, frequency, sound_to_use = echo_sound, distance_multiplier = 0)
 
 			if(base_shake_amount || quake_factor)
-				base_shake_amount = max(base_shake_amount, quake_factor * 3, 0) // Devastating explosions rock the station and ground
+				base_shake_amount = max(base_shake_amount, quake_factor * 6, 0) // Devastating explosions rock the station and ground
```

Comment:
This is borderline creating an accessibility issue, as myself, my husband, and like 4 of my friends that routinely play this game that I know of alone get motion sickness and/or headaches from shit moving around violently randomly. Would you mind making this a scaling factor set in preferences instead of a set value so people don't literally have to stop playing because of this change?